lti_consumer/__init__.py

```diff
@@ -4,4 +4,4 @@
 from .apps import LTIConsumerApp
 from .lti_xblock import LtiConsumerXBlock
 
-__version__ = '4.1.0'
+__version__ = '4.2.0'
```

Comment:
1. Let's not update version (or changelogs) for any PRs with these changes. I don't think it is necessary.
2. Can you add the following to commit messages:
```
Rename toggle_warnings to toggle_warning for consistency
with setting_warning.
```
3. Can you add that same sentence, as well as a link to the code-annotations PR, to all the PR descriptions?

Thank you.

Reply:
Ok, got it